internal/client/util.go

```diff
@@ -133,7 +133,9 @@ func request(ctx context.Context, client *http.Client, cfg requestConfig) (*http
 	}
 
 	if !success {
-		return nil, fmt.Errorf("status code %s", resp.Status)
+		body, _ := io.ReadAll(resp.Body)
+
+		return nil, fmt.Errorf("status code=%s, error=%s", resp.Status, body)
```

Comment:
resolves https://linear.app/prefect/issue/PLA-559/http-error-message-sometimes-not-propagated

Reply:
```
│ Could not create deployment, unexpected error: failed to create deployment: status code=422 Unprocessable Entity, error={"exception_message":"Invalid request
│ received.","exception_detail":[{"type":"greater_than","loc":["body","concurrency_limit"],"msg":"Input should be greater than
│ 0","input":0,"ctx":{"gt":0}}],"request_body":{"concurrency_limit":0,"flow_id":"9b824122-bb76-43c3-aded-4a629b11b76c","name":"my-deployment"}}
```

Reply:
🤦🏼 oops. great find dude, really glad to have the full error message back